executable_command.go

```diff
@@ -214,10 +214,10 @@ func getHelpFromMagicComments(reader *bufio.Reader) (string, error) {
 
 func getMessageFromExecution(path string, flag string) (string, error) {
 	cmd := exec.Command(path, "--"+flag)
-	out, err := cmd.CombinedOutput()
+	cmd.Stderr = nil
+	out, err := cmd.Output()
```

Comment:
Yep, I had completely read the previous line as though it was returning (stdout, stderr), _not_ as the actual (merged stdout + stderr, Go error)

That'll do it!